sonic_platform_base/sonic_storage/storage_base.py

```diff
@@ -1,11 +1,11 @@
 #
-# ssd_base.py
+# storage_base.py
 #
 # Base class for implementing common SSD health features
```

Comment:
@assrinivasan comment does not look correct

Reply:
Changed in latest